peachjam/templates/peachjam/_months_list.html

```diff
@@ -0,0 +1,35 @@
+{% load i18n %}
+{% if years %}
+  {% if showallmonths %}
+    <div>
+      <a class="{% if not month %} active{% endif %}"
+         href="{{ all_months_url }}">{% trans 'All months' %}</a>
+    </div>
+  {% endif %}
+  <div class="pl-0 mt-2 mb-3 align-items-center d-none d-lg-flex">
+    <i class="bi bi-calendar me-2"></i>
+    <ul class="year-nav mb-0 ms-2">
+      {% for m in months %}
+        <li>
+          <a class="{% if m.month == month %} active{% endif %}"
```

Comment:
if `m.month == month` then just render `m.month` without a tag around it.

Reply:
By rendering without a tag, we'll lose the url.